docs/attributes-registry/cicd.md

```diff
@@ -28,3 +43,21 @@ This group describes attributes specific to pipelines within a Continuous Integr
 | `build` | build | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `deploy` | deploy | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `test` | test | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+---
+
+`cicd.worker.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `busy` | The worker is performing work for the CICD system. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `idle` | The worker is not performing work for the CICD system. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+---
+
+`cicd.worker.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
I think we should add `container`

Reply:
Does it make sense to distinguish between pod and container?

Reply:
A pod can have several containers so a pod is a wrapper around containers.

Reply:
Argo Workflows can run a DAG inside a [Container Set](https://argo-workflows.readthedocs.io/en/latest/container-set-template/) so in this case the number of containers and number of pods would not match each other.
These containers can be `busy` and `idle` and switch state, so this has meaning here.
(Not: the conventional ("one pod for one job") pod spun up by Argo would actually have two containers (the `main` doing the work and `wait` which is the controller), although I'm not sure it would be useful to track that in the metrics.

Reply:
I added `container` as an additional value in 66732c5f627a60e5ea1d0f337b0914ce33f1ebeb.